pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConnection.java

```diff
@@ -458,25 +459,6 @@ protected void handleLookup(CommandLookupTopic lookup) {
         lookupProxyHandler.handleLookup(lookup);
     }
 
-    private synchronized void close() {
-        if (state != State.Closed) {
-            state = State.Closed;
-            if (directProxyHandler != null && directProxyHandler.outboundChannel != null) {
-                directProxyHandler.outboundChannel.close();
-                directProxyHandler = null;
-            }
-            if (connectionPool != null) {
-                try {
-                    connectionPool.close();
```

Comment:
I was wondering if we needed to call close here, but looks like that'll happen in the `channelInactive` method that netty will call.

Reply:
Yes, I removed it since there was duplication. the `channelInactive` method contains the cleanup.